gatsby-config.js

```diff
@@ -26,27 +28,41 @@ module.exports = {
             baseId: `appNYMWxGF1jMaf5V`,
             tableName: `Organizations`,
             tableView: `Published`,
-            mapping: { "Published": `boolean`, "Logo": `fileNode` },
-            tableLinks: [`Sector`]
+            mapping: { Published: `boolean`, Logo: `fileNode` },
+            tableLinks: [`Sector`],
           },
           {
             baseId: `appNYMWxGF1jMaf5V`,
             tableName: `Sectors`,
             tableView: `Published`,
-            mapping: { "Cover": `fileNode` },
-            tableLinks: [`Organizations`]
-          }
-        ]
-      }
+            mapping: { Cover: `fileNode` },
+            tableLinks: [`Organizations`],
+          },
+        ],
+      },
     },
     `gatsby-plugin-react-helmet`,
     {
       resolve: `gatsby-transformer-sharp`,
       options: {
         defautQuality: 75,
-      }
+      },
     },
     `gatsby-plugin-sharp`,
-    `gatsby-plugin-postcss`
+    `gatsby-plugin-postcss`,
   ],
 }
+
+if (process.env.ALGOLIA_ADMIN_KEY) {
```

Comment:
There are 3 new algolia keys that are needed. I am not exactly sure how we'll share those internally for testing, but it should be doable